docs/tutorials/getting-started.ipynb

```diff
@@ -0,0 +1,469 @@
+    "length / speed"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e47256a2-0bac-4457-a8a7-d97f998758c8",
+   "metadata": {},
+   "source": [
+    "The unit is an invaluable safeguard against errors:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dad2fcef-4825-447e-9c65-aaf792960750",
+   "metadata": {
+    "tags": [
+     "raises-exception"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "length + speed  # will raise an exception"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ba67dc9-bb7c-4fc4-9718-90aef3f097f8",
+   "metadata": {},
+   "source": [
+    "With the above we can use scipp as a safer version of a \"calculator\"."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85832186-c294-4976-a9e0-ece74e7be4a0",
+   "metadata": {},
+   "source": [
+    "## Physical Constants\n",
+    "\n",
+    "Scipp provides physical constants in the [scipp.constants](https://scipp.github.io/generated/modules/scipp.constants.html) module.\n",
+    "These are taken from SciPy but are represented as variables with a unit, for ease of use and safety.\n",
+    "We may, e.g., import the speed of light $c$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a47e549-9e09-4ee0-a4ee-2283b5a99a37",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from scipp.constants import c"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7fc7010f-5b11-47c8-bc4f-9b0607ec39c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b9852c2b-16ac-4d5d-9184-48df4e2a8e27",
+   "metadata": {},
+   "source": [
+    "This can be used with the quantities we defined above:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "83c94310-7de3-428a-9445-0818d7eba867",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "length / c"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a98e32e1-9d18-4adc-aa2c-7a8987627131",
+   "metadata": {},
+   "source": [
+    "## Uncertainties\n",
+    "\n",
+    "Scipp operations perform linear error propagation, but **operations do not track correlations**.\n",
+    "Please **carefully consider** whether this is appropriate for your application! \n",
+    "Otherwise the error bars that scipp computes are highly misleading as they give a false sense of security but may actually be completely wrong.\n",
+    "See the [uncertainties](https://pythonhosted.org/uncertainties/) Python package for a solution that *can* track correlations.\n",
+    "\n",
+    "To create a quantity with variance we use the \"creation function\" `scalar`, which returns a variable:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b6bc810c-ec39-4c9a-901f-b66d88a40782",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "length = sc.scalar(50.0, variance=0.01, unit='mm')\n",
```

Comment:
I think it's a little confusing to people who read this for the first time that we were using `1.2 * sc.Unit('m')` above, and `sc.scalar` here. I suggest to only use `sc.scalar()`, and maybe mention as a side note, e.g. after the first use of `scalar`, that you can also use the `*` syntax as an alternative.

I also think that many python beginners will struggle to fully grasp what `sc.Unit('m')` is actually doing/representing.

Reply:
hmm maybe `quantity` is better?